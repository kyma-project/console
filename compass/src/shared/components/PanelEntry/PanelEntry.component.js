import { Panel } from '@kyma-project/react-components';
import React from 'react';
import PropTypes from 'prop-types';

const PanelEntry = ({ title, content }) => (
  <Panel>
    <Panel.Body>
      <p className="fd-has-color-text-4 fd-has-margin-bottom-none">{title}</p>
      {content}
    </Panel.Body>
  </Panel>
);

PanelEntry.propTypes = {
  title: PropTypes.string.isRequired,
  content: PropTypes.node.isRequired,
};

export default PanelEntry;
