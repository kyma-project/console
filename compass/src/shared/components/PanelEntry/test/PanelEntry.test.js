import React from 'react';
import renderer from 'react-test-renderer';
import PanelEntry from '../PanelEntry.component';

describe('PanelEntry', () => {
  it(`Renders title and content`, () => {
    const component = renderer.create(
      <PanelEntry title="testtitle" content="testcontent" />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
