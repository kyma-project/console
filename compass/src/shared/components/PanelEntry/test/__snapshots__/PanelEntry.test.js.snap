// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelEntry Renders title and content 1`] = `
<div
  className="fd-panel"
>
  <div
    className="fd-panel__body"
  >
    <p
      className="fd-has-color-text-4 fd-has-margin-bottom-none"
    >
      testtitle
    </p>
    testcontent
  </div>
</div>
`;
